### PR TITLE
HAWKULAR-609 - Implemented the 'Export JDR' operation.

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

**Comment by jpkrohling** [comment]
@mtho11 , could you please review this one?

File: src/rest/hawkRest-ops-factory.ts [hunk 1]
```diff
@@ -58,7 +65,7 @@ module hawkularRest {
 
   interface IWebSocketResponseHandler {
     prefix: string;
-    handle: (any) => void;
+    handle: (any, binaryData:Blob) => void;
```
Comment (mtho11) [inline src/rest/hawkRest-ops-factory.ts]: should the binaryData be optional? binaryData?:Blob instead?

Comment (jpkrohling) [inline src/rest/hawkRest-ops-factory.ts]: +1 , updated.

**Comment by mtho11** [comment]
Merged #46

**Pull request merged**